docs/commands/bundle.md

```diff
@@ -1,117 +1,91 @@
 # `bundle`
 
+## Introduction
+
 API definitions can grow and become difficult to manage, especially if several teams are collaborating on them. It's a good practice to maintain the reusable parts as separate files, and include them in the main (root) API definition by referencing them with `$ref`. However, most OpenAPI tools do not support that multi-file approach, and require a single-file API definition.
 
 Redocly OpenAPI CLI can help you combine separate API definition files into one. The `bundle` command pulls the relevant parts of an API definition into a single file output in JSON or YAML format.
 
 The `bundle` command first executes preprocessors, then rules, then decorators.
 
-```mermaid
-graph LR
-    preprocessors[Preprocessors]
-    rules[Rules]
-    decorators[Decorators]
-    preprocessors --> rules --> decorators
-```
-
-### `bundle` usage
-
-```
-Positionals:
-  entrypoints                                              [array] [default: []]
-Options:
-  --version            Show version number.                            [boolean]
-  --help               Show help.                                      [boolean]
-  --lint               Lint definitions               [boolean] [default: false]
-  --output, -o                                                          [string]
-  --format             Use a specific output format.
-                        [choices: "stylish", "codeframe"] [default: "codeframe"]
-  --max-problems       Reduce output to max N problems.  [number] [default: 100]
-  --ext                Bundle file extension.   [choices: "json", "yaml", "yml"]
-  --skip-rule          Ignore certain rules.                             [array]
-  --skip-preprocessor  Ignore certain preprocessors.                     [array]
-  --skip-decorator     Ignore certain decorators.                        [array]
-  --dereferenced, -d   Produce fully dereferenced bundle.              [boolean]
-  --force, -f          Produce bundle output even when errors occur.   [boolean]
-  --config             Specify path to the config file.                 [string]
-```
+:::success Tip
+To learn more about preprocessors, rules, and decorators, please refer to [this](../custom-rules.md) document.
```

Comment:
suggestion: Can you please change the link anchor text to be more descriptive instead of just "this"? Something like "refer to the [custom rules page](link)" would be fine.